retroshare-gui/src/gui/feeds/ChatMsgItem.cpp

```diff
@@ -36,6 +36,9 @@
 
 #include "gui/msgs/MessageInterface.h"
 
+#include "util/qtthreadsutils.h"
+#include <retroshare/rsevents.h>

```

Comment:
<> is for system includes, and "" for local includes. Not a problem, but just for you to know.

Reply:
I will fix it and another one in SecurityIpitem.h